### PR TITLE
OSDOCS-6090: GCP CCM is GA (RNs)

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.15

Issue:
[OSDOCS-6090](https://issues.redhat.com//browse/OSDOCS-6090) > [OSDOCS-6091](https://issues.redhat.com//browse/OSDOCS-6091)

Link to docs preview:
* [Cloud controller managers for additional cloud providers](https://69888--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-cluster-cloud-controller-manager-operator)
* [Machine management Technology Preview features](https://69888--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#machine-management-technology-preview-features)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@JoelSpeed here are the RN updates for GCP CCM GA, PTAL

**Review by JoelSpeed — COMMENTED** [review]
LGTM

**Comment by jeana-redhat** [comment]
@miyadav here are the rel notes related to the other PR, PTAL

**Comment by miyadav** [comment]
thanks @jeana-redhat , LGTM

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -430,6 +430,19 @@ The Network Observability Operator releases updates independently from the {prod
...
+
+The Kubernetes community plans to deprecate the use of the Kubernetes controller manager to interact with underlying cloud platforms in favor of using cloud controller managers. As a result, there is no plan to add Kubernetes controller manager support for any new cloud platforms.
+
+This release introduces the General Availability of using a cloud controller manager for Google Cloud Platform.
+
+To learn more about the cloud controller manager, see the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Kubernetes Cloud Controller Manager documentation].
+
+To manage the cloud controller manager and cloud node manager deployments and lifecycles, use the Cluster Cloud Controller Manager Operator.
+
+For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Platform Operators reference_.
```
Comment (skopacz1) [inline release_notes/ocp-4-15-release-notes.adoc:444]: The xref points to a page titled Cluster Operators reference, should that be the text in the italics here?
```suggestion
For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Cluster Operators reference_.
```

**Review by skopacz1 — COMMENTED** [review]
Just one comment, otherwise looks good to me!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-15-release-notes.adoc:444]: That's what I get for copy-pasting from last version. Thanks!

**Pull request merged**